Dockerfile

```diff
@@ -2,8 +2,9 @@ FROM cm2network/steamcmd:root
 
 RUN set -x \
     && apt-get update \
-    && DEBIAN_FRONTEND=noninteractive apt-get install -y sudo=1.8.27-1+deb10u3 --no-install-recommends\
-    && rm -rf /var/lib/apt/lists/*
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y gosu --no-install-recommends\
```

Comment:
regarding [DL3008](https://github.com/wolveix/satisfactory-server/runs/4409009169?check_suite_focus=true) I'd ignore this rule in this case:
1. as long as the image is based on debian stable it won't upgrade the version anyway beyond security fixes (which we probably want).
2. we shouldn't pin the apt version if we don't / can't `FROM` a tag with a fixed debian version. a future image update may not have the pinned version available.

Reply:
Agreed :)